tox.ini

```diff
@@ -33,11 +31,13 @@ setenv =
     py39: INTERP=py39
     pypy3: INTERP=pypy3
     nocov: PYTHONASYNCIODEBUG=1
-    !py39: PY_39=lt
-    py39: PY_39=ge
+    py36: PY_36=eq
+    !py36: PY_36=gt
     PLATFORM={env:PLATFORM:linux}
 passenv =
     PYTHON*
+    TRAVIS
+    CI
```

Comment:
These env-vars signals `housekeep.py` to skip some housekeeping on ephemeral testing VMs (Travis CI, Github Actions, etc.)